doc/quickstart/re3gistry_quickstart.rst

```diff
@@ -0,0 +1,104 @@
+
+If this is the first start, the Re3gistry will launch the installation wizard. This wizard will guide the user to the setup and first run of the tool.
+
+
+* The first page is just an introductory description. Click on "continue".
+* The "Registry manager" page allows you to define the account for the system administration (Registry manager). Insert a valid e-mail address and a password. Then click on "Save and continue".
+* The next step allows you to choose between 2 options:
+  * Installation: choose this option if you want to install the Re3gistry software for the first time and to create a registry from scratch.
+  * Migration: choose this option if you want to upgrade a pre-existing Re3gistry instance. This option will keep your settings and content. Note that this option will only work on an existing instance running on Re3gistry version 1.3.X.
+
+**Installation option**
+
+In case the "Installation" option is chosen, the basic configuration for the new registry will be requested.
+
+* System languages: the system languages are the languages supported and handled by the Re3gistry 2. Note that you can select one or more system languages.
+* Master language: the master language is the one used by the Re3gistry 2 to handle multilingualism. Note that this will require all content to be available in the chosen master language.
+* Base URI: the base URI will be part of the public URL, generally this should match with your own domain name: **http://example.com**/registry
+* Local ID: the local ID will be part of the public URL, generally the path following the domain name: http://example.com/**registry**
+* Label: the label of the registry will be the ‘title’ of the Re3gistry 2 instance. It will be shown in the registry’s landing page.
+* Content summary: this text should shortly describe what would be available under your Re3gistry 2 instance. It will be shown in the registry’s landing page.
+
+After filling all the information, click on "Save and continue".
+
+The next step will present a summary of the information inserted. If all the fields are correct, click on "Install" to begin the installation.
+Once the installation is finished, the user is redirected to the login page.
+
+**Migration option**
+
+In case the "Migration" option is chosen, the details of the database from the existing Re3gistry installation are requested.
+Fill the requested fields and click the "Save and continue" button. You will then get a page containing a summary of the data retrieved from the existing instance of the Re3gistry.
+Click on "Install" to begin the migration. An information page will be shown. Now you can close the browser and the system will notify you via e-mail using the address specified in the "Registry manager" definition step.
+
+Introduction to the Re3gistry 2
+================
```

Comment:
Make 80 chars long preferably but at least make it at least as long as the title